package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "soundclout-active-player-module",
+  "version": "1.0.0",
+  "description": "This features an animated player synced proportionally with a time-stamped location in a song. Its clickable components include a play/pause toggle button, fastforward/rewind buttons, shuffle/repeat buttons, hover volume control, album thumbnails, a like button, and a \"up next\" pop up menu.  The \"up next\" pop up menu also features a button to clear the list, a toggle to autoplay upcoming tracks, as well as like buttons and a pop-up options menu for each track.  The menu allows you to like, share, add or remove from \"next up\", add to a playlist, or start a radio station of related tracks.",
```

Comment:
Consider shortening this description in future PRs, as this is quite verbose. Full details should more likely be placed into this repo's README file.

Reply:
Git init pulled it directly from the README description

Reply:
I shortened the description to the first sentence.